File: C/Ex-33_Page-142.c
//
//  Ex-33_Page-142.c
//  Exercise 33 Page 142
//  29042023
//
//  Created by Sebastian Jusca on 28/03/2023.
//

#include <stdio.h>

int main()
{
    float price, discPrice;
    
    printf("Enter your total: ");
    scanf("%f",& price);
    
    if(price < 100){
        discPrice = price * 0.05;
    }else if (price >= 200){
        discPrice = price * 0.15;}
    else{
        discPrice = price * 0.1;
    }
    
    price = price - discPrice;
    
    printf("Your discounted price is %f €\n", price);
    
    return 0;
}

File: Python/Ex-33_Page-142.py
# 
# Ex-33_Page-142.py
# Exercise 33 Page 142
# a6a32a119163eefe77e05f6dc6c17e585b4f4ee8
#
# Created by Sebastian Jusca on 17/04/2023.
#

price = float(input('Enter your total: '))

if price < 100:
	discPrice = price * 0.5
elif price >= 200:
	discPrice = price * 0.15
else:
	discPrice = price * 0.1

price = price - discPrice

print('Your discounted price is {}'.format(price))